src/mainwindow.cpp

```diff
@@ -1131,7 +1137,7 @@ void MainWindow::createHelpMenu()
                    fileName);
         continue;
       }
-      QAction* tutAction = new QAction(params.at(0), tutorialMenu);
+      QAction* tutAction = new QAction(tr(translate[params.at(0)]), tutorialMenu);
```

Comment:
I don't understand why you would need the `tr` here if you already use the `QT_TR_NOOP` above?